### DISCUSSION TITLE
iTunes

### DISCUSSION THREAD

**Discussion by Desmay2** [FrostCo/AdvancedProfanityFilter#470] [Ideas]
This may be an absolutely stupid question, but is there a way of using this in iTunes? Downloading it as a plugin and adding it to iTunes?

**Comment by richardfrost** [comment]
Not a dumb question @Desmay2, but unfortunately since this is only a browser extension that's the only place it can work. Let me know if you have any other questions though!

**Reply by Desmay2** [reply]
Another thing - it doesn't seem to work with Kindle (read.amazon.com) for some reason?

**Reply by richardfrost** [reply]
@Desmay2 It was first reported in issue #348, but unfortunately things haven't improved yet as far as I know. To summarize it, they made a change to deliver the book text as an image now, which the filter doesn't work on. Hopefully I can work out something though, and I'll add it to my to do list to take another look at it to see if the situation has improved at all.

**Reply by Desmay2** [reply]
> @Desmay2 It was first reported in issue #348, but unfortunately things haven't improved yet as far as I know. To summarize it, they made a change to deliver the book text as an image now, which the filter doesn't work on. Hopefully I can work out something though, and I'll add it to my to do list to take another look at it to see if the situation has improved at all.

Wow that was fast. Thank you very much anyway :) Google eBooks seems to work still which is good.

**Comment by Desmay2** [comment]
No worries, thank you very much!

Regards,
Xander
